akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala

```diff
@@ -575,7 +575,7 @@ abstract class ActorContextSpec extends ScalaTestWithActorTestKit(
       }.decorate)
       val adapterName = "hello"
       actor ! adapterName
-      val adapter = probe.expectMessageType[ActorRef[String]]
+      val adapter = probe.receiveOne()
```

Comment:
👍 These changes to the new receiveOne are great, much cleaner, thanks for doing it.